qui/tray/updates.py

```diff
@@ -89,15 +90,38 @@ def __init__(self, app_name, qapp, dispatcher):
         self.tray_menu = Gtk.Menu()
         self.fullscreen_window_hack.show_for_widget(self.tray_menu)
 
+        # For those who never restart their PC, update EOL notifs every 6 hours
+        GLib.timeout_add_seconds(3600 * 6, self.check_vms_needing_update)
+
     def run(self):  # pylint: disable=arguments-differ
         self.check_vms_needing_update()
         self.connect_events()
 
         self.update_indicator_state()
 
+    def obsolete_dom0(self, _event):
+        open_url_in_disposable(
+            "https://www.qubes-os.org/doc/supported-releases/",
+            self.qapp,
+        )
+
     def setup_menu(self):
         self.tray_menu.set_reserve_toggle_size(False)
 
+        if "dom0" in self.obsolete_vms:
+            self.tray_menu.append(
+                RunItem(
+                    _(
+                        f"<span foreground=\"red\"><b>"
+                        "Warning!!! "
```

Comment:
While emotionally I'm all for triple exclamation mark here, I think one would look more serious :)